msal/build.gradle

```diff
@@ -239,6 +239,8 @@ dependencies {
     String common4jVersion = project.hasProperty("distCommon4jVersion") ? project.distCommon4jVersion : "0.0.+"
     testDistImplementation(testFixtures("com.microsoft.identity:common4j:${common4jVersion}"))
 
+    implementation platform("io.opentelemetry:opentelemetry-bom:1.18.0")
```

Comment:
nit: The version numbers can go in versions.gradle file, to make it easier to update in one place.

Reply:
I will handle all these version numbers in separate PR.